translations/en/LC_MESSAGES/messages.po

```diff
@@ -19,49 +19,49 @@ msgstr ""
 
 msgid "Cyclic Var Definition"
-msgstr "We detected that a variable `{variable}` is being used on the right-hand side of the `{is}` command before being set. Can you set the variable before it is used?"
+msgstr "We detected that a variable `{variable}` is being used on the right-hand side of the `{is}` before being set. Can you set the variable first?"
 
 msgid "Else Without If Error"
 msgstr "We detected that there is an `{else}` being used before an `{if}` on line {line_number}. Can you try writing an `{if}` before the `{else}`?"
 
 msgid "Function Undefined"
-msgstr "We detected that a function {name} is being used without being defined. Can you define the function before it is used?"
+msgstr "We detected that a function `{name}` is being used without being defined. Can you define the function first?"
 
 msgid "Has Blanks"
 msgstr "We detected that the code seems to be incomplete. Can you fill in the blanks with code?"
 
 msgid "Incomplete"
-msgstr "We detected that part of the code seems to be missing from the `{incomplete_command}` on line {line_number}. Can you try adding the what's missing?"
+msgstr "We detected that part of the code seems to be missing from the `{incomplete_command}`, on line {line_number}. Can you try adding what's missing?"
 
 msgid "Incomplete Repeat"
-msgstr "We detected that the `{repeat}` on line {line_number} is missing a `{command}` command. Can you try adding it?"
+msgstr "We detected that the `{repeat}` seems to be missing a `{command}` command, on line {line_number}. Can you try adding `{repeat}` with `{command}`?"
 
 msgid "Invalid"
 msgstr "We detected that `{invalid_command}` is not a Hedy level {level} command. Can you try using `{guessed_command}`?"
 
 msgid "Invalid Argument"
-msgstr "We detected that `{command}` is not usable with `{invalid_argument}`. Can you try changing `{invalid_argument}` to {allowed_types}?"
+msgstr "We detected that `{command}` may not be used with `{invalid_argument}`. Can you try changing `{invalid_argument}` to {allowed_types}?"
 
 msgid "Invalid Argument Type"
-msgstr "We detected that `{command}` doesn't work with `{invalid_argument}` because it is {invalid_type}. Can you try changing `{invalid_argument}` to {allowed_types}?"
+msgstr "We detected that `{command}` may not be used with `{invalid_argument}` because it is {invalid_type}. Can you try changing `{invalid_argument}` to {allowed_types}?"
 
 msgid "Invalid At Command"
-msgstr "We detected that `{command}` may not be used from level 16 onward. Can you try using square brackets `[]` for lists?"
+msgstr "We detected that `{command}` may not be used from level 16 onward. Can you try adding square brackets around `[]` the list?\""
 
 msgid "Invalid Space"
-msgstr "We detected that line {line_number} started with a space. Can you try removing the space?"
+msgstr "We detected that line {line_number} started with a space. Can you remove the space?"
```

Comment:
Here we are removing try, which is think is good, but it still is there in other places.